api/src/main/java/com/example/api/model/activity/result/GraphTaskResult.java

```diff
@@ -53,6 +54,7 @@ public GraphTaskResult(GraphTask graphTask,
         this.graphTask = graphTask;
         this.setUser(user);
         this.startDateMillis = startDateMillis;
+        this.sendDateMillis = startDateMillis;
```

Comment:
Jest jakiś powód ustawiania tego pola, bo widze w kodzie, że przynajmniej w jednym miejscu jest sprawdzane czy graphTask jest ukończony przez studenta właśnie na podstawie tego czy sendDateMillis nie jest nullem

Reply:
Wcześniej żadna z ekspedycji po przejściu przez nowe flow nie była widziana, i to jest workaround - spójrz na dyskusję w https://github.com/ajablonsk1/systematic-chaos/pull/191#discussion_r979218217 i jeśli uda się to zakodować, to będzie można wyrzucić na luzie

Reply:
zmieniło się na setter, ale logika stojąca za tym jest taka sama

Reply:
Jak dla mnie - powinien zostać ustawiony ten sendDate na start tak jak jest, lepsze to niż null tutaj a nie ma żadnych przeciwwskazań żeby na początek było == startDate